docs/getting-started.md

```diff
@@ -14,6 +15,10 @@ PayID is a simple, web-based protocol designed to make it as easy to send someon
 
 Your contributions are welcome. Check out the [PayID repository on GitHub](https://github.com/payid-org/payid), and note the [Contributing](https://github.com/payid-org/payid/blob/master/CONTRIBUTING.md) guidelines.
 
+## Experiment with the PayID Sandbox
+
+The [PayID Sandbox](https://payid.org/sandbox) lets you experiment with PayID. Log in with your Github account and follow the prompts. You can quickly set up a virtual server, and then create users with simple PayID addresses that map to addresses on various networks such as XRPL, Interledger, BTC, and ETH. Using the provided cURL or Javascript code, you can create, update, and delete users, and get payment information about a PayID address. While the session is running, you can also run operations from your command line. Your work is not saved between sessions, and the sandbox may be reset at any time.  
```

Comment:
Maybe be a little more explicit in the first two sentences that the sandbox is for experimenting with PayID in a test environment, and explicitly not for running a production PayID instance?

Reply:
Updated this section to make that clearer.